woocommerce/woocommerce-gutenberg-products-block
# Move `SortSelect` to components package

## What

Move `SortSelect` component to components package and update the places it is imported:

- `ProductSortSelect`
- `ReviewSortSelectProps`

## Why


As more developers integrate their extensions with WooCommerce Blocks, the case for reusing components grows. This starts the ball rolling by creating a new package. Rather than recreating components developers can access `wc.blocksComponents` and get the ones we need.


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Set up some reviews on your products (Go to the product on the front end then the Reviews tab)
2. Add the All Reviews and All Products blocks to a page.
3. Visit that page and ensure the select box to change the ordering of reviews is visible. Interact with it and ensure the order of the reviews changes.
4. On the All Products block, ensure the select box to change the ordering of products is visible. Interact with it and ensure the order of products changes.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Make the `SortSelect` component available in the `@woocommerce/blocks-components` package.
